pkg/network/ebpf/c/http-types.h

```diff
@@ -8,7 +8,7 @@
 // This controls the number of HTTP transactions read from userspace at a time
 #define HTTP_BATCH_SIZE 15
 // The greater this number is the less likely are colisions/data-races between the flushes
-#define HTTP_BATCH_PAGES 15
+#define HTTP_BATCH_PAGES 5
```

Comment:
Why is this decreasing? According to the comment, making it smaller is a "bad" thing.